examples/pytorch/pointcloud/pointnet/pointnet2.py

```diff
@@ -114,6 +114,22 @@ def forward(self, edges):
             res = pos
         return {'agg_feat': res}
 
+class RelativePositionMessageMSG(nn.Module):
```

Comment:
Why do we need a version for MSG?

Reply:
Referenced from https://github.com/yanx27/Pointnet_Pointnet2_pytorch/blob/master/models/pointnet_util.py , not mentioned in the paper

Reply:
My question is essentially: why do we need a `RelativePositionMessageMSG`, if we already have a `RelativePositionMessage`?

Reply:
Slightly different. The orders of the arguments are somehow switched in their implementations.

Reply:
What impact does it have, and can we just adapt to the original implmentation?

Reply:
Deleted the MSG version.